packages/wcg-core/platforms/ios/WinterCG.xcframework/ios-arm64/WinterCG.framework/Source/NSCBase64.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  NSCBase64.swift
+//  WinterCG
+//
+//  Created by Osei Fortune on 09/01/2024.
+//
+
+import Foundation
+
+@objcMembers
+@objc(NSCBase64)
+public class NSCBase64: NSObject {
+    public static func atob(_ value: String) -> String {
+        let decoded = wcg_core_atob(value.cString(using: .utf8))
+        if(decoded == nil){
+            return ""
+        }
```

Comment:
Could turn this into a `guard let`. I'm writing without an IDE here (so don't accept the suggestion blindly, but something like this:

```suggestion
        guard let decoded = wcg_core_atob(value.cString(using: .utf8), decoded != nil) { else return "" }
```

There are some other identical cases below, too.